docs/0-what-is/1-getting-started.mdx

```diff
@@ -5,81 +5,183 @@ sidebar_position: 0
 slug: '/getting-started'
 ---
 
-Pipeline configuration files give you the ability to define pipelines that are
-provisioned by Conduit at startup. It's as simple as creating a YAML file that
-defines pipelines, connectors, processors, and their corresponding
-configurations.
+## Our goal
 
-## Example pipeline
+In this guide, we'll learn how to install Conduit and use it Conduit to build
+and run a pipeline. The pipeline will be continuously generating sample user
+data (using the
+built-in [generator connector](https://github.com/ConduitIO/conduit-connector-generator))
+and printing the data to logs. The user data will contain an ID (which is an
+integer) and a name (which is a string).
 
-:::tip
+## Install Conduit
 
-In our [Conduit repository](https://github.com/ConduitIO/conduit), you can find [more examples](https://github.com/ConduitIO/conduit/tree/main/examples/pipelines), but to ilustrate a simple use case we'll show a pipeline using a file as a source, and another file as a destination. Check out the different [specifications](/docs/using/pipelines/configuration-file) to see the different configuration options.
+The easiest way to get started with Conduit on your macOS or Linux system is to
+install it using our installation script. Simply run the following command
+in your terminal:
 
+```shell
+$ curl https://conduit.io/install.sh | bash
+```
+
+If you're not using macOS or Linux system, you can still install Conduit
+following one of the different options provided
+in [our installation page](/docs/installing-and-running).
+
+:::note
+The Conduit binary contains both, the Conduit service and the Conduit CLI, with
+which you can interact with Conduit.
 :::
 
-Create a folder called `pipelines` at the same level as your Conduit binary.
-Inside of that folder create a file named `file-to-file.yml`.
+## Initialize Conduit 
```

Comment:
Since this is a quick start, we might even want to skip this part. It's optional anyway, and we can cut the time to getting the pipeline running.

Reply:
I would leave it with just the option of `conduit init` to be honest. I think it's a good idea to guide them through that since we have that command available. It'll help with educating them about where things are.